packages/docs-app/src/examples/core-examples/collapsePlaygroundExample.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from "react";
+
+import { Button, Collapse, H5, Pre, Switch } from "@blueprintjs/core";
+import { Example, type ExampleProps, handleBooleanChange } from "@blueprintjs/docs-theme";
+
+export const CollapsePlaygroundExample: React.FC<ExampleProps> = props => {
```

Comment:
Would be nice to update this example to make the keep children mounted toggle actually make a difference. As it stands currently, this toggle appears to have no effect on the example.

Reply:
Hm, I'm open to ideas on how best to demo it. IMO, the new "Keeping children mounted" section demonstrates this pretty well with the form input state either being mounted/unmounted.

I've added an input field to the playground example to the same effect: f625df3062f0c8cc477f4ff2e73fb0118e20086f

Let me know if that seems adequate!

Reply:
Yeah, that should do the trick just fine.